### PR TITLE
Add trace for regisering vsync call

### PR DESCRIPTION
**Pull request by Shao-Feng**
Change-Id: I509ec86a36630ee70f917a9b0f4d7908b330a4d2
Signed-off-by: Shaofeng Tang <shaofeng.tang@intel.com>

### CODE REVIEW

File: common/display/vblankeventhandler.h [hunk 1]
```diff
@@ -27,8 +27,6 @@
 
 #include "hwcthread.h"
 
-#define VBLANK_THRESHHOLD 2777777
```
Comment (YuanjunHuang) [inline common/display/vblankeventhandler.h:30]: what does this threshold mean here?

Comment (Shao-Feng) [reply common/display/vblankeventhandler.h:30]: We ever think only when the different in vperiod is larger than this one, the 2.4 callback should be invoked.
For instance, when the period is changed from 90hz to 60hz

File: os/android/iahwc2.cpp [hunk 2]
```diff
@@ -344,6 +344,7 @@ HWC2::Error IAHWC2::RegisterCallback(int32_t descriptor,
       for (size_t i = 0; i < size; ++i) {
         IAHWC2::HwcDisplay *display = extended_displays_.at(i).get();
         display->RegisterVsyncCallback(data, function);
+        ETRACE("######Register original Vsync callback");
```
Comment (YuanjunHuang) [inline os/android/iahwc2.cpp:347]: If this aims to check in, do not use etrace for log only info.

Comment (Shao-Feng) [reply os/android/iahwc2.cpp:347]: just for verifying, I will commit another one once this one work

**Pull request merged**